### DISCUSSION TITLE
Tools :: Store Manager: What's the sense of the 'Reset ALL Products Ordered"?

### DISCUSSION THREAD

**Discussion by lat9** [zencart/zencart#6116] [General]
Clicking the associated button resets the `products::products_ordered` field for all products, but **_Reports :: Products Purchased_** counts based on the `orders_products` table.

It looks like the primary use of that database field is in determining the storefront best-sellers.  One of my clients (as did I) had the expectation that the "Reset" would also reset the output of that report.

**Comment by drbyte** [comment]
Correct: it resets the `products_ordered` field.

_(edit)_
That resets the [`bestsellers` sidebox](https://github.com/zencart/zencart/blob/0bcbca5cffd5834cd29205680ab3a58dd58d50f8/includes/modules/sideboxes/best_sellers.php#L29)

**Reply by drbyte** [reply]
As for the the`Products Purchased` report, I think "at one point" we might have based that report on products_ordered but found it less reliable or unperformant. I remember we had several different SQL queries in the report's code that people could uncomment to swap out the calculation based on their needs. I know we had that for a few reports. I think that was one of them.
